<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>1141</width>
      <height>661</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::West</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <property name="documentMode">
     <bool>true</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>选择目标</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>841</width>
        <height>71</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="inputLabel">
         <property name="text">
          <string>输入图片：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="inputlineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="browseButton">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(186, 189, 182);</string>
         </property>
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="selectROIButton">
         <property name="text">
          <string>选择目标</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>570</y>
        <width>841</width>
        <height>71</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="outputLabel">
         <property name="text">
          <string>输出图片：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="outputlineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="browseButton_2">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(186, 189, 182);</string>
         </property>
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>100</y>
        <width>241</width>
        <height>461</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="pluginsLayout">
       <item>
        <widget class="QLabel" name="pluginsLabel">
         <property name="text">
          <string>插件</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="pluginsList"/>
       </item>
       <item>
        <widget class="QPushButton" name="pluginsButton">
         <property name="text">
          <string>应用</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="helpButton">
         <property name="text">
          <string>帮助</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>100</y>
        <width>581</width>
        <height>451</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="imageLabel">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab1">
     <attribute name="title">
      <string>检索模式</string>
     </attribute>
     <widget class="QPushButton" name="videoInButton">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>600</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>打开</string>
      </property>
     </widget>
     <widget class="QPushButton" name="videoProcessButton">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>600</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>检索</string>
      </property>
     </widget>
     <widget class="QPushButton" name="videoInitButton">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>600</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>载入</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>921</width>
        <height>461</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="searchLabel4">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="searchLabel3">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="searchLabel2">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="searchLabel1">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab2">
     <attribute name="title">
      <string>地图模式</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab3">
     <attribute name="title">
      <string>多机模式</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>1061</width>
        <height>521</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="video1">
           <property name="text">
            <string>Video1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="video2">
           <property name="text">
            <string>Video2</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="onButton">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>570</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>打开</string>
      </property>
     </widget>
     <widget class="QPushButton" name="offButton">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>610</y>
        <width>201</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>关闭</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>运行</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>导入</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
